resources/certificate.rb

```diff
@@ -49,55 +57,52 @@
   equal_to: %w(smtp pop3 imap ftp xmpp xmpp-server irc postgres mysql lmtp nntp sieve ldap),
   description: 'A protocol specific STARTTLS argument to use when fetching from an ssl_endpoint'
 
+property :file_cache_path, String,
```

Comment:
I don't love this property name; any better suggestions?

Reply:
I can't think of anything better.